doc/nrf/libraries/gazell/gzp.rst

```diff
@@ -10,9 +10,55 @@ Gazell Pairing
 The Gazell pairing library enables applications to use the Gazell Link Layer to provide a secure wireless link between Gazell nodes.
 The library is customized for pairing a device (for example, a mouse, keyboard, or remote control) with a host (typically a USB dongle) using Gazell.
 
-See the :ref:`ug_gzp` user guide for more information.
+Overview
+********
+See the :ref:`ug_gzp` user guide for more information, such as the features of this library.
 
-This library is used in the :ref:`gzp_dynamic_pairing` sample.
+This library is used in the :ref:`gzp_dynamic_pairing_host` and :ref:`gzp_dynamic_pairing_device` samples.
+
+Requirements
+************
+
+In addition to the resources required by the :ref:`ug_gzll` (see :ref:`ug_gzll_resources`), the Gazell Pairing library also employs three nRF5 peripherals:
+
+* Random Number Generator, for generating keys and tokens.
+* AES Electronic Codebook (ECB), for encryption and decryption.
+* Non-Volatile Memory Controller (NVMC), for storing of pairing parameters.
+
+In addition, Gazell Pairing employs the following Gazell Link Layer resources:
+
+* Two pipes: one for pairing and one for encrypted data transmission.
+* Gazell Pairing determines the channel set used by Gazell.
+
+Since Gazell Pairing requires exclusive access to pipes 0 and :c:macro:`GZP_DATA_PIPE` (default pipe 1), it must control the internal Gazell Link Layer variables ``base_address_0``, ``base_address_1`` and ``prefix_address_byte`` for pipes :c:macro:`GZP_PAIRING_PIPE` (always pipe 0) and :c:macro:`GZP_DATA_PIPE` (configurable).
+The main application can use the pipes 2-7.
+The ``base_address_1`` applies to these pipes.
+Gazell Pairing must also determine whether the RX pipes 0 and 1 are enabled.
+Make sure not to affect the ``rx_enabled`` status of these pipes.
+
+The following Gazell Link Layer API functions should not be accessed:
+
+* :c:func:`nrf_gzll_set_base_address_0()`
+* :c:func:`nrf_gzll_set_base_address_1()`
+* :c:func:`nrf_gzll_set_address_prefix_byte()` (for pipes 0 and 1)
```

Comment:
Is this `for pipes 0 and 1` equal to "this is required for pipes 0 and 1" or "this should not be accessed for pipes 0 and 1"?

Reply:
I will ask and discuss with @grochu

Reply:
The intention I think is: "When the Gazell Pairing Library is used, the `nrf_gzll_set_address_prefix_byte()` API must not be called for pipes 0 and 1".

Reply:
Okay, thanks! I will add a few words to this line to make it even more clear.